packages/editor/src/functions/browserCheck.ts

```diff
@@ -24,13 +24,13 @@ Infinite Reality Engine. All Rights Reserved.
 */
 
 export const isSupportedBrowser = () => {
-  const userAgent = navigator.userAgent as any
+  const userAgent = window.navigator.userAgent
   const winNav = window.navigator as any
 
   const isGoogleChrome =
     /Chrome/.test(userAgent) && typeof winNav.userAgentData !== 'undefined'
       ? winNav.userAgentData.brands?.some((item) => item.brand === 'Google Chrome')
```

Comment:
@aditya-mitra I think we can keep this here since we refactor this so it wont crash the app if `brands` is null. I added an extra step to check for a other chromium browsers to be safe. 

Let me know if you would prefer to remove the use of useAgentData completly although I think it's good to have for now

Reply:
`userAgentData` is unavailable in a lot of brands
we can avoid the ternary conditions (separate testing again for safari) and for maintenance purposes just narrow it down to `window.navigator.userAgent`

Reply:
cool, updated